tests/performance/load-tests/load-test.sh

```diff
@@ -63,6 +80,17 @@ function parseArguments() {
   done
 }
 
+function cleanup() {
+  echo "Clean up the environment"
+
+  if [ $start_separately = true ]; then
+    echo "Delete test namespaces"
+    kubectl delete namespace --selector=test-type=load-tests >/dev/null 2>&1 || true
```

Comment:
can we add selector definition to the top of the script ?

Reply:
Ok